Looks like someone deleted the README file out of the output directory again. Can you readd that?

```
Error: main.go:18:12: pattern web/tanzu-ui/build: no matching files found (typecheck)
```

--- a/cli/cmd/plugin/ui/server/serve.go
+++ b/cli/cmd/plugin/ui/server/serve.go
@@ -81,9 +81,7 @@ func Serve(bind, browser string, logLevel int32) error {
 // apiMiddleware routes the API request handling
 func apiMiddleware(handler http.Handler) http.Handler {
 	return requestLogger(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		if strings.HasPrefix(r.URL.Path, "/ws") {
-			handlers.HandleWebsocketRequest(w, r)
-		} else if strings.HasPrefix(r.URL.Path, "/api") {
+		if strings.HasPrefix(r.URL.Path, "/api") {
 			handler.ServeHTTP(w, r)
 		} else {
 			http.Redirect(w, r, "/ui", http.StatusMovedPermanently)
@@ -100,7 +98,9 @@ func globalMiddleware(handler http.Handler) http.Handler {
 
 func fileServerMiddleware(next http.Handler) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		if !strings.HasPrefix(r.URL.Path, "/ui") {
+        if strings.HasPrefix(r.URL.Path, "/ws") {
+    		handlers.HandleWebsocketRequest(w, r)
+        } else if !strings.HasPrefix(r.URL.Path, "/ui") {
 			if r.URL.Path == "/" {
 				http.Redirect(w, r, "/ui", http.StatusMovedPermanently)
 			} else {

--- a/cli/cmd/plugin/ui/server/serve.go
+++ b/cli/cmd/plugin/ui/server/serve.go
@@ -98,8 +98,8 @@ func globalMiddleware(handler http.Handler) http.Handler {
 
 func fileServerMiddleware(next http.Handler) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-        if strings.HasPrefix(r.URL.Path, "/ws") {
-    		handlers.HandleWebsocketRequest(w, r)
+	    if strings.HasPrefix(r.URL.Path, "/ws") {
+            handlers.HandleWebsocketRequest(w, r)
         } else if !strings.HasPrefix(r.URL.Path, "/ui") {
 			if r.URL.Path == "/" {
 				http.Redirect(w, r, "/ui", http.StatusMovedPermanently)

--- a/cli/cmd/plugin/ui/server/serve.go
+++ b/cli/cmd/plugin/ui/server/serve.go
@@ -98,9 +98,9 @@ func globalMiddleware(handler http.Handler) http.Handler {
 
 func fileServerMiddleware(next http.Handler) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-	    if strings.HasPrefix(r.URL.Path, "/ws") {
-            handlers.HandleWebsocketRequest(w, r)
-        } else if !strings.HasPrefix(r.URL.Path, "/ui") {
+		if strings.HasPrefix(r.URL.Path, "/ws") {
+			handlers.HandleWebsocketRequest(w, r)
+		} else if !strings.HasPrefix(r.URL.Path, "/ui") {
 			if r.URL.Path == "/" {
 				http.Redirect(w, r, "/ui", http.StatusMovedPermanently)
 			} else {

